Benchmark.jl

```diff
@@ -0,0 +1,12 @@
+using MLJTime
+
+const univariate_data = univariate_datasets
+
+TSdataset(univariate_data)
+
+
+function Benchmake_all(algorithms)
```

Comment:
I wouldn't include unfinished functions in a PR

Reply:
Oh, That function is added by mistake!